docker/web/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
+MAINTAINER Pedro Marques <roque@juniper.net>
+RUN apt-get install -y software-properties-common
+RUN apt-add-repository -y ppa:anantha-l/opencontrail-2.20
```

Comment:
Good catch, thanks. At the time, Vinay had not uploaded web packages to
opencontrail ppa. So I was using mine.. I have updated pull request with
this.

On Wed, Oct 28, 2015 at 4:40 AM, Pedro Marques <notifications@github.com>
wrote:

> In docker/web/Dockerfile
> <https://github.com/pedro-r-marques/examples/pull/1#discussion_r43200793>:
>
> > @@ -0,0 +1,29 @@
> > +FROM ubuntu:14.04
> > +MAINTAINER Pedro Marques <roque@juniper.net>
> > +RUN apt-get install -y software-properties-common
> > +RUN apt-add-repository -y ppa:anantha-l/opencontrail-2.20
>
> Why do we need your personal repo ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pedro-r-marques/examples/pull/1/files#r43200793>.
>